src/herder/test/HerderTests.cpp

```diff
@@ -6684,3 +6684,65 @@ TEST_CASE("trigger next ledger side effects", "[herder][parallel]")
     REQUIRE(herder.getTriggerTimer().seq() > 0);
     REQUIRE(herder.mTriggerNextLedgerSeq == nextSeq + 2);
 }
+
+TEST_CASE("detect dead nodes in quorum set", "[herder]")
+{
+
+    Hash networkID = sha256(getTestConfig().NETWORK_PASSPHRASE);
+    Simulation::pointer simulation = Topologies::core(
+        3, 0.5, Simulation::OVER_LOOPBACK, networkID, [&](int i) {
+            auto cfg = getTestConfig(i, Config::TESTDB_DEFAULT);
+            // We set SET_CLOSE_TIME_FOR_TESTING to shorten the testing period
+            cfg.ARTIFICIALLY_SET_CLOSE_TIME_FOR_TESTING = 5;
```

Comment:
I don't think this config is needed.

Reply:
Hm, something doesn't sound right here. This is virtual time, so none of these configs should matter. If you want 5 seconds close time, just remove `ARTIFICIALLY_SET_CLOSE_TIME_FOR_TESTING` and `ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING` completely (which will fallback to the default 5 second ledgers).